### DISCUSSION TITLE
点歌: diffusion model 相关，如Diffusion Models Beat GANs on Image Synthesis,尤其是基本原理，如何train/infer

### DISCUSSION THREAD

**Discussion by xv44586** [mli/paper-reading#157] [General]
[Diffusion Models Beat GANs on Image Synthesis](https://arxiv.org/pdf/2105.05233.pdf)

**Comment by Aitical** [comment]
+++++++++++1

**Comment by BEaN001** [comment]
++++++

**Comment by zmzhang** [comment]
可以做一个 diffusion models 论文串烧，包括：
[Denoising Diffusion Probabilistic Models](https://arxiv.org/abs/2006.11239)
[Diffusion Models Beat GANs on Image Synthesis](https://arxiv.org/abs/2105.05233)
[Cascaded Diffusion Models for High Fidelity Image Generation](https://arxiv.org/abs/2106.15282)
[GLIDE: Towards Photorealistic Image Generation and Editing with Text-Guided Diffusion Models](https://arxiv.org/abs/2112.10741)

**Reply by zzhouj** [reply]
High-Resolution Image Synthesis with Latent Diffusion Models [https://arxiv.org/abs/2112.10752]

**Comment by zhouliang-yu** [comment]
yeaaaah, diffusion seems to be a super popular recently haha

**Comment by mli** [comment]
加了三篇在TODO里，感觉这一块需要补很多背景知识

**Reply by Eric123007** [reply]
期待老师的diffusion系列！

**Reply by xuzhang5788** [reply]
++++++++++++++++1
